applications/dashboard/settings/class.hooks.php

```diff
@@ -725,7 +725,7 @@ public function rootController_sso_create($Sender, $Target = '') {
 
         // If an event handler didn't handle the signin then just redirect to the target.
         if (!$Handled) {
-            safeRedirect($Target, 302);
+            redirectTo($Target, 302);
```

Comment:
I know you just switched over the functions, but 302 is the default response code, so you shouldn't need to explicitly state it here.